#request
bool mode
uint8 pinNumber
---
#response
string str
